src/navigator-card-body.jsx

```diff
@@ -391,7 +391,7 @@ export const NavigatorCardBody = ({
         );
 
     const contextMenu = (
-        <ContextMenu parentId="folder-view">
+        <ContextMenu parentId="navigator-card">
```

Comment:
This fix is alright, but this really advocates for moving the whole `<Card` into a new component class. But that's follow up territory.